src/lelantus.cpp

```diff
@@ -1284,9 +1259,8 @@ int CLelantusState::GetCoinSetForSpend(
                 blockHash_out = block->GetBlockHash();
             }
             numberOfCoins += block->lelantusMintedPubCoins[id].size();
-            coins_out.insert(coins_out.end(),
-                block->lelantusMintedPubCoins[id].begin(),
-                block->lelantusMintedPubCoins[id].end());
+            for(const auto& coin : block->lelantusMintedPubCoins[id])
```

Comment:
`operator []` on map creates an empty element if it doesn't exist. Need to check if it exists before accessing